doc/source/math/direction.rst

```diff
@@ -185,6 +185,13 @@ Structure
 
     :struct:`Vector` of length 1 that is in the same direction as the "starboard side" of this Direction.  Note that it is the same meaning as "what the X axis of the universe would be rotated to if this rotation was applied to the basis axes of the universe". When you LOCK STEERING to a direction, that direction's STARVECTOR is the vector the right wing of the ship will orient to.  SHIP:FACING:STARVECTOR is the way the ship's right wing is aimed right now.
 
+.. attribute:: Direction:INVERSE
+
+    :type: :struct:`Direction`
+    :access: Get only
+    
+    :struct: Gives a `Direction` with the opposite rotation around its axes. So `r(30,15,5)` becomes `r(-30,-15,-5)`.
+    
```

Comment:
just warped round kerbin running this script:
```
Until false {
 set dir1 to vecdrawargs(v(0,0,0),r(30,15,5):inverse:vector,red,"Dir1",5,true).
 set dir2 to vecdrawargs(v(0,0,0),r(-30,-15,-5):vector,green,"Dir2",5,true).
}.
```
Both arrows stayed over lapping the whole way round.